docs/contribute/development-environment.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Saros Development Environment
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+The following page describes how a development environment can be set up in order to develop all Saros products (Eclipse-Plugin, IntelliJ-Plugin and Server).
+If you want to execute the STF tests it is recommended to use eclipse. Otherwise it is also possible to develop with IntelliJ IDEA and execute the stf tests in a docker container. 
+
+## Common
+* You have to [clone](https://help.github.com/articles/cloning-a-repository/) the Saros repository with [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+* You need a **Java 8 JDK**.
+* You need an **Eclipse 4.6** installation which is used for dependency resolution. You can either install a [minimal Eclipse](http://www.eclipse.org/downloads/packages/release/neon/3/eclipse-ide-java-developers) or [Eclipse for Eclipse Committers](http://www.eclipse.org/downloads/packages/release/neon/3/eclipse-ide-eclipse-committers) if you also want to develop with eclipse.
+* Install [GEF Legacy](https://projects.eclipse.org/projects/tools.gef/downloads) into Eclipse
+* You need an **Intellij IDEA** installation which is used for dependency resolution. Install a current version of [**IntelliJ IDEA**](https://www.jetbrains.com/idea/download/#section=linux) (we have only tested Saros/I for IntelliJ releases 2017.X and later)
+
+Set the **system-wide environment variable `ECLIPSE_HOME`** to the eclipse installation dir that contains the directory `plugins`.
+Set the **system-wide environment variable `INTELLIJ_HOME`** to the intellij installation dir that contains the directory `lib`.
+
+If the `ECLIPSE_HOME` variable is not set or not global the correponding eclipse specific dependencies cannot be found during build.
+If the `INTELLIJ_HOME` variable is not set or not global the intellij-gradle plugin will download a version of IntelliJ which is
+defined in the gradle build description.
+
+## Develop with Eclipse
+If you develop on Eclipse you should have already installed the Eclipse version for "Eclipse Committers".
+### Import the Saros Project
+* Open a bash terminal, navigate to the repository directory and execute `./gradlew prepareEclipse` (use `./gradlew.bat` for windows)
+* Import the project as Git project
+* If you add dependencies you have to execute the `prepareEclipse` task again in order to regenerate the dependency information for eclipse.
+
+## Develop with Intellij
+It is necessary to import the saros project and change the project setting so that all build/test/debug actions are processed
+by Gradle.
+
+### Import Project
+* Click on `Import Project`
+* Select the repository root as project root and click `OK`
+* Select `Import project from external model`, select `Gradle` and click `Next`
```

Comment:
Importing the project will overwrite the existing .idea folder. Since we are distributing this folder, you have to just open the project in IntelliJ (instead of importing it).

Reply:
Done